README.md

```diff
@@ -4,41 +4,41 @@
+
+Use Materialize to do things like deliver fresh context for AI/RAG pipelines, power operational dashboards, and create more dynamic customer experiences without building time-consuming custom data pipelines.
+
+
+The three most common patterns for adopting Materialize are the following:
+
+- Query Offload (CQRS) - Scale complex read queries more efficiently than a read replica, and without the headaches of cache invalidation.
+- Integration Hub (ODS) - Extract, load, and incrementally transform data from multiple sources. Create live views of your data that can be queried directly or pushed downstream.
+- Operational Data Mesh (ODM) - Use SQL to create and deliver real-time, strongly consistent data products to streamline coordination across services and domains.
 
-Using SQL and common tools in the wider data ecosystem, Materialize allows you
-to build real-time automation, engaging customer experiences, and interactive
-data products that drive value for your business while reducing the cost of
-[data freshness](https://materialize.com/blog/decouple-cost-and-freshness/).
 
 ## Get started
 
-Ready to try out Materialize? [Sign up](https://materialize.com/register/) to get started! 🚀
+Ready to try out Materialize? You can [sign up](https://materialize.com/register/) for a free cloud trial or [download](https://materialize.com/download/) our community edition, which is free forever for deployments using less than 24 GiB of memory and 48 GiB of disk!
 
 Have questions? We'd love to hear from you:
   * [Join our Slack](https://materialize.com/s/chat)
   * [Send us mail](https://materialize.com/contact/)
 
 ## About
 
-Materialize is designed to help you interactively explore your streaming data, perform analytics against live relational data, or increase data freshness while reducing the load of your dashboard and monitoring tasks. The moment you need a refreshed answer, you can get it in milliseconds.
-
-It focuses on providing correct and [consistent](https://materialize.com/docs/overview/isolation-level/) answers with minimal latency, and does not ask you to accept either approximate answers or eventual consistency. Whenever Materialize answers a query, that answer is the correct result on some specific (and recent) version of your data. Materialize does all of this by recasting your SQL queries as *dataflows*, which can react efficiently to changes in your data as they happen.
+Materialize focuses on providing correct and [consistent](https://materialize.com/docs/overview/isolation-level/) answers with minimal latency, and does not ask you to accept either approximate answers or eventual consistency. Whenever Materialize answers a query, that answer is the correct result on some specific (and recent) version of your data. Materialize does all of this by recasting your SQL queries as *dataflows*, which can react efficiently to changes in your data as they happen.
```

Comment:
```suggestion
Materialize focuses on providing correct and [consistent](https://materialize.com/docs/overview/isolation-level/) answers with minimal latency, and does not ask you to accept either approximate answers or eventual consistency. This guarantee holds even when joining data from [multiple upstream systems](https://materialize.com/blog/strong-consistency-in-materialize/). Whenever Materialize answers a query, that answer is the correct result on some specific (and recent) version of your data. Materialize does all of this by recasting your SQL queries as *dataflows*, which can react efficiently to changes in your data as they happen.
```

Reply:
Multi-system consistency is always a wow moment and something that I'd suggest more directly calling out.